siwe/siwe.py

```diff
@@ -177,13 +175,25 @@ def prepare_message(self) -> str:
         return "\n\n".join([prefix, suffix])
 
     def to_message(self) -> str:
-        warnings.warn("deprecated", DeprecationWarning)
+        warnings.warn("deprecated, use prepare_message()", DeprecationWarning)
         return self.prepare_message()
 
     def sign_message(self) -> str:
-        warnings.warn("deprecated", DeprecationWarning)
+        warnings.warn("deprecated, use prepare_message", DeprecationWarning)
         return self.prepare_message()
 
+    def fields(self) -> Dict:
```

Comment:
the use of `__slots__` overrides the default `__dict__` function and prevents it from being created (https://docs.python.org/3/reference/datamodel.html#slots)

```
> msg = siwe.SiweMessage(tmp['message'])
> msg.__dict__
AttributeError                            Traceback (most recent call last)
Input In [4], in <cell line: 1>()
----> 1 msg.__dict__

AttributeError: 'SiweMessage' object has no attribute '__dict__'
```

But I hear you on the rest. I'll extract this function to be used in my code, rather than in the core library.